src/tmle_inputs/from_actors.jl

```diff
@@ -148,22 +166,29 @@ function get_variables(pcs, traits, extraW, extraC, extraT)
     )
 end
 
-function parameters_from_actors(bqtls, transactors, data, variables, orders, outprefix; positivity_constraint=0., batch_size=nothing)
+function parameters_from_actors(bqtls, transactors, data, variables, orders, outprefix, tf_name; positivity_constraint=0., batch_size=nothing)
     parameters = TMLE.Parameter[]
     batch_id = 1
     extraT_df = variables.extra_treatments isa Nothing ? nothing : DataFrame(ID=variables.extra_treatments)
     # For each interaction order generate parameter files
     for order in orders
         # First generate the `T` section
         treatment_combinations = TargeneCore.combine_by_bqtl(bqtls, transactors, extraT_df, order)
+        # If there are duplicates here, remove them
+        treatment_combinations = unique(treatment_combinations)
```

Comment:
Is the previous function generating duplicates?

Reply:
This can happen if different types of trans-actors for the same TF share the same RSID.

Reply:
Hum, let's discuss about that